snuba/replacer.py

```diff
@@ -190,7 +191,7 @@ def process_delete_groups(message):
 SEEN_MERGE_TXN_CACHE = deque(maxlen=100)
 
 
-def process_merge(message):
+def process_merge(message, all_column_names):
```

Comment:
Seems like these should eventually be moved somewhere other than module scope. (I'm assuming that's on the roadmap for some later point?)

Reply:
yes, it is going away, but the replacer will be the last step of the migration.